x-pack/test/functional/apps/lens/dashboard.ts

```diff
@@ -116,6 +121,18 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       );
       const hasGeoDestFilter = await filterBar.hasFilter('geo.dest', 'LS');
       expect(hasGeoDestFilter).to.be(true);
+      await filterBar.addFilter('geo.src', 'is', 'US');
+      await filterBar.toggleFilterPinned('geo.src');
```

Comment:
In terms of code style, I'm not a big fan of how the next text depends on test setup from this test. Could you move this logic into a single test?